### PR TITLE
Cloud iam user access investigation xsiam

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-2750)

## Description
New playbook "Cloud IAM User Access Investigation" to investigate and respond XCloud alerts where a Cloud IAM users access key is used suspiciously to access the cloud environment.
Playbooks `AWS IAM User Access Investigation` and `AWS IAM User Access Investigation and remediation` - deprecated.

## Screenshots
Cloud_IAM_User_Access_Investigation

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [X] 6.6.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* core-get-cloud-original-alerts
+* closeInvestigation
+
+## Playbook Inputs
+
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| autoAccessKeyRemediation | Whether to execute the user remediation flow automatically. | False | Optional |
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md:37]: The user or the user's access key?

Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md:37]: fixed to user

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 2]
```diff
@@ -0,0 +1,888 @@
...
+      URL:
+        complex:
+          root: alert
+          accessor: url
+      User:
+        complex:
+          root: alert
+          accessor: username
+      query:
+        simple: sourceBrand:"${alert.sourceBrand}" and name:"${alert.name}"
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:593]: Does the query is good enough for checking previous alerts?

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 3]
```diff
@@ -0,0 +1,888 @@
...
+      CloseReason:
+        simple: Resolved - False Positive,Resolved - Duplicate Incident,Resolved - Known Issue
+      Domain:
+        complex:
+          root: alert
+          accessor: domainname
+      FileSHA256:
+        complex:
+          root: alert
+          accessor: initiatorsha256
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:579]: let's use playbook inputs for sha256

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 4]
```diff
@@ -0,0 +1,888 @@
...
+      IP:
+        complex:
+          root: alert
+          accessor: hostip
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:583]: let's use playbook inputs for host ip

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 5]
```diff
@@ -0,0 +1,888 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "66":
+    id: "66"
+    taskid: cfb35229-4702-4fa4-8b8b-f9d86778c329
+    type: condition
+    task:
+      id: cfb35229-4702-4fa4-8b8b-f9d86778c329
+      version: -1
+      name: Malicious?
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:394]: Give more meaningful title

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 6]
```diff
@@ -0,0 +1,888 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 08cf3974-ddc8-4694-8a1a-e737b3e69973
+    type: condition
+    task:
+      id: 08cf3974-ddc8-4694-8a1a-e737b3e69973
+      version: -1
+      name: Manual decision making - true/false-positive alert
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:110]: Maybe "Manual verdict decision" ? or something like that .shorter

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 2 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:593]: I think the Host IP and the username should be a part of the query.

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 3 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:579]: I removed it from the task.

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 4 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:583]: In the playbook on XSOAR, Yarden told me to remove this input and change it to incident.hostip

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 5 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:394]: Done.

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 6 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:110]: Done

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 4 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:579]: OK, if Yarden wants to use less inputs than keep the alert.x keys.

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 7]
```diff
@@ -0,0 +1,900 @@
...
+              simple: alert.name
+            iscontext: true
+          right:
+            value:
+              simple: Suspicious API call from a Tor exit node
+        - operator: isEqualNumber
+          left:
+            value:
+              complex:
+                root: DBotScore
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:435]: Add the filter we've talked about.
where DBotScore.Indicator eq alert.hostip

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 8]
```diff
@@ -0,0 +1,900 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "66":
+    id: "66"
+    taskid: cfb35229-4702-4fa4-8b8b-f9d86778c329
+    type: condition
+    task:
+      id: cfb35229-4702-4fa4-8b8b-f9d86778c329
+      version: -1
+      name: Is it malicious activity?
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:409]: a malicious activity

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 9]
```diff
@@ -0,0 +1,900 @@
...
+      description: Is it a suspicious or Tor IP?
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:410]: unclear description

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 10]
```diff
@@ -0,0 +1,900 @@
...
+  "73":
+    id: "73"
+    taskid: 6e5634a7-7393-4cfa-8bb4-b2beca7a782e
+    type: regular
+    task:
+      id: 6e5634a7-7393-4cfa-8bb4-b2beca7a782e
+      version: -1
+      name: Investigate collected data
+      description: |2-
+         To determine if this is a true positive event,  review the operations performed by the access key and the user in the recent time frame.
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:539]: Fix the syntax to be presented in a comfort way for reading using markdown.

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 11]
```diff
@@ -0,0 +1,900 @@
...
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 08cf3974-ddc8-4694-8a1a-e737b3e69973
+    type: condition
+    task:
+      id: 08cf3974-ddc8-4694-8a1a-e737b3e69973
+      version: -1
+      name: Manual verdict decision
+      description: Based on the collected data investigation, is it a true positive event?
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:111]: Based on....., select the verdict of the alert.

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 12]
```diff
@@ -0,0 +1,900 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: 043b96c8-7870-4754-823d-3a0658690cf2
+    type: title
+    task:
+      id: 043b96c8-7870-4754-823d-3a0658690cf2
+      version: -1
+      name: False Positive - Done
```
Comment (melamedbn) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:143]: Why do we need two 'Done' headers if one is connected to the other.

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 9 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:410]: Done

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 10 above)
Comment (OmriItzhak) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:539]: Done

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 9 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:410]: @OmriItzhak 
Checks wether the alert is malicious based on either one of the following:
1. Source IP address with a malicious DBot score.
2. A Tor IP address.

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml (hunk 10 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:539]: image
Doesn't look so good.

**Review by melamedbn — APPROVED** [review]

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 13]
```diff
@@ -0,0 +1,907 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "66":
+    id: "66"
+    taskid: cfb35229-4702-4fa4-8b8b-f9d86778c329
+    type: condition
+    task:
+      id: cfb35229-4702-4fa4-8b8b-f9d86778c329
+      version: -1
+      name: Is it malicious activity?
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:409]:
```suggestion
      name: Is activity malicious?
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 14]
```diff
@@ -0,0 +1,907 @@
...
+    isautoswitchedtoquietmode: false
+  "67":
+    id: "67"
+    taskid: 4f102212-5337-40d7-8477-f983500618fe
+    type: regular
+    task:
+      id: 4f102212-5337-40d7-8477-f983500618fe
+      version: -1
+      name: Load alert JSON
+      description: Loads a json from string input, and returns a json object result
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:476]:
```suggestion
      description: Loads a JSON from a string input, and returns a JSON object.
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 15]
```diff
@@ -0,0 +1,907 @@
...
+  "73":
+    id: "73"
+    taskid: 6e5634a7-7393-4cfa-8bb4-b2beca7a782e
+    type: regular
+    task:
+      id: 6e5634a7-7393-4cfa-8bb4-b2beca7a782e
+      version: -1
+      name: Investigate collected data
+      description: ' To determine if this is a true positive event,  review the operations performed by the access key and the user in the recent time frame.<br>Search for the following items:<br>-Investigate the operation performed by the access key and examine the executed operations, by who it was executed, on which resource, and the operation status.<br>-Investigate operations performed by the user and examine the executed operations, by who it was executed, on which resource, and the operation status.<br>-Look at any persistence, for example - a new user creation or key, etc.<br>-Investigate operations performed by the user and examine the executed operations, by who it was executed, on which resource, and the operation status.<br>-Look at any lateral movement operations. For example, an operation can be = AsumeRole.<br>As an extra validation step, it is recommended to query the user and/or the user’s manager regarding the investigated suspicious activity'
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:549]:
```suggestion
      description: 'To determine if this is a true positive event, review the operations performed by the access key and the user in the recent time frame.<br>Search for the following items:<br>-Investigate the operation performed by the access key and examine the executed operations, by who it was executed, on which resource, and the operation status.<br>-Investigate operations performed by the user and examine the executed operations, by who it was executed, on which resource, and the operation status.<br>-Look at any persistence, for example - a new user creation or key, etc.<br>-Look at any lateral movement operations. For example, an operation can be = AsumeRole.<br>As an extra validation step, it is recommended to query the user and/or the user’s manager regarding the investigated suspicious activity.'
```

(The removed line was repeated.)

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 16]
```diff
@@ -0,0 +1,907 @@
...
+    isautoswitchedtoquietmode: false
+  "80":
+    id: "80"
+    taskid: 3e6e0a3b-c353-46a8-895c-e46d5fcb4efe
+    type: playbook
+    task:
+      id: 3e6e0a3b-c353-46a8-895c-e46d5fcb4efe
+      version: -1
+      name: Enrichment for Verdict
+      description: This playbook checks prior alert closing reasons and performs enrichment and prevalence checks on different IOC types. It then  returns the information needed to establish the alert's verdict.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:580]:
```suggestion
      description: This playbook checks prior alert closing reasons and performs enrichment and prevalence checks on different IOC types. It then returns the information needed to establish the alert's verdict.
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 17]
```diff
@@ -0,0 +1,907 @@
...
+- key: GCP-accessKeyRemediationType
+  value:
+    simple: Disable
+  required: false
+  description: |-
+    Choose the remediation type for the user's access key.
+
+    GCP available types:
+    Disable - For disabling the user's access key.
+    Delete - For the deleting user's access key.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:884]:
```suggestion
    Delete - For deleting the user's access key.
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 18]
```diff
@@ -0,0 +1,907 @@
...
+- key: AWS-accessKeyRemediationType
+  value:
+    simple: Disable
+  required: false
+  description: |-
+    Choose the remediation type for the user's access key.
+
+    AWS available types:
+    Disable - for disabling the user's access key.
+    Delete - for the user's access key deletion.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:851]:
```suggestion
    Delete - for deleting the user's access key.
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 19]
```diff
@@ -0,0 +1,907 @@
...
+  playbookInputQuery:
+- key: AWS-userRemediationType
+  value:
+    simple: Revoke
+  required: false
+  description: |-
+    Choose the remediation type for the user involved.
+
+    AWS available types:
+    Delete - for the user deletion.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:861]:
```suggestion
    Delete - for deleting the user.
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md [hunk 20]
```diff
@@ -0,0 +1,56 @@
...
+### Sub-playbooks
+
+* Handle False Positive Alerts
+* Cloud IAM Enrichment - Generic
+* Cloud Response - Generic
+* Enrichment for Verdict
+
+### Integrations
+
+* CortexCoreIR
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md:20]:
```suggestion
CortexCoreIR
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md [hunk 21]
```diff
@@ -0,0 +1,56 @@
...
+
+### Scripts
+
+* LoadJSON
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md:24]:
```suggestion
LoadJSON
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md [hunk 22]
```diff
@@ -0,0 +1,56 @@
...
+## Playbook Inputs
+
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| autoAccessKeyRemediation | Whether to execute the user remediation flow automatically. | False | Optional |
+| autoBlockIndicators | Whether to block the indicators automatically. | False | Optional |
+| autoUserRemediation | Whether to execute the user remediation flow automatically. | False | Optional |
+| AWS-accessKeyRemediationType | Choose the remediation type for the user's access key.<br/><br/>AWS available types:<br/>Disable - for disabling the user's access key.<br/>Delete - for the user's access key deletion. | Disable | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md:40]:
```suggestion
| AWS-accessKeyRemediationType | Choose the remediation type for the user's access key.<br/><br/>AWS available types:<br/>Disable - for disabling the user's access key.<br/>Delete - for deleting the user's access key. | Disable | Optional |
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+| AWS-userRemediationType | Choose the remediation type for the user involved.<br/><br/>AWS available types:<br/>Delete - for the user deletion.<br/>Revoke - for revoking the user's credentials. | Revoke | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md:41]:
```suggestion
| AWS-userRemediationType | Choose the remediation type for the user involved.<br/><br/>AWS available types:<br/>Delete - for deleting the user.<br/>Revoke - for revoking the user's credentials. | Revoke | Optional |
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md [hunk 24]
```diff
@@ -0,0 +1,56 @@
...
+| Azure-userRemediationType | Choose the remediation type for the user involved.<br/><br/>Azure available types:<br/>Disable - for disabling the user.<br/>Delete - for deleting the user. | Disable | Optional |
+| GCP-accessKeyRemediationType | Choose the remediation type for the user's access key.<br/><br/>GCP available types:<br/>Disable - For disabling the user's access key.<br/>Delete - For the deleting user's access key. | Disable | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md:43]:
```suggestion
| GCP-accessKeyRemediationType | Choose the remediation type for the user's access key.<br/><br/>GCP available types:<br/>Disable - For disabling the user's access key.<br/>Delete - For deleting the user's access key. | Disable | Optional |
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md [hunk 25]
```diff
@@ -0,0 +1,56 @@
...
+| GCP-userRemediationType | Choose the remediation type for the user involved.<br/><br/>GCP available types:<br/>Delete - For deleting the user.<br/>Disable - For disabling the user. | Disable | Optional |
+| ShouldCloseAutomatically | Whether to close alerts automatically as a false positive \(True/False\). | False | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation_README.md:45]:
```suggestion
| ShouldCloseAutomatically | Whether to close alerts automatically as a false positive. \(True/False\). | False | Optional |
```

File: Packs/Core/ReleaseNotes/1_3_43.md [hunk 26]
```diff
@@ -0,0 +1,21 @@
+
+#### Playbooks
+##### AWS IAM User Access Investigation
+
+- Deprecated. Use `Cloud IAM User Access Investigation` instead.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/1_3_43.md:5]:
```suggestion
Deprecated. Use `Cloud IAM User Access Investigation` instead.
```

File: Packs/Core/ReleaseNotes/1_3_43.md [hunk 27]
```diff
@@ -0,0 +1,21 @@
...
+
+##### AWS IAM User Access Investigation - Remediation
+
+- Deprecated. Use `Cloud IAM User Access Investigation` instead.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/1_3_43.md:9]:
```suggestion
Deprecated. Use `Cloud IAM User Access Investigation` instead.
```

File: Packs/Core/ReleaseNotes/1_3_43.md [hunk 28]
```diff
@@ -0,0 +1,21 @@
...
+
+- Investigate and respond to Cortex XSIAM alerts where a Cloud IAM user access key is used suspiciously to access the cloud environment.
+The following alerts are supported for AWS, Azure, and GCP environments.
+Penetration testing tool attempt
+Penetration testing tool activity
+Suspicious API call from a Tor exit node (Available from Cortex XSOAR 6.8.0).
+
+#### Triggers Recommendations
+
+- New: **Cloud IAM User Access Investigation**
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/1_3_43.md:21]:
```suggestion
New: **Cloud IAM User Access Investigation**
```

File: Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml [hunk 29]
```diff
@@ -0,0 +1,907 @@
...
+
+    GCP available types:
+    Delete - For deleting the user.
+    Disable - For disabling the user.
+  playbookInputQuery:
+- key: ShouldCloseAutomatically
+  value:
+    simple: "False"
+  required: false
+  description: Whether to close alerts automatically as a false positive (True/False).
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml:901]:
```suggestion
  description: Whether to close alerts automatically as a false positive. (True/False).
```

**Comment by ShirleyDenkberg** [comment]
@melamedbn Doc review completed.

**Pull request merged**